templates/aws/shared/_get_pro.html

```diff
@@ -5,13 +5,19 @@ <h2>Get Ubuntu Pro on AWS now:</h2>
     </div>
   </div>
   <div class="row p-divider">
-    <div class="col-4 p-divider__block">
+    <div class="col-2 p-divider__block">
+      <h3 class="p-heading--4"><a href="https://aws.amazon.com/marketplace/pp/prodview-s4zvkzmlirbga">24.04 LTS</a></h3>
```

Comment:
```suggestion
      <h3 class="p-heading--4"><a href="https://aws.amazon.com/marketplace/pp/prodview-kbkyvwyqubv2g">24.04 LTS</a></h3>
```
The link is different from the copy doc